command-expansion-server/Dockerfile.dev

```diff
@@ -0,0 +1,4 @@
+FROM node:lts-alpine
+#COPY . /app
```

Comment:
Is this commented since it's used during test sometimes?

Reply:
I commented this simply because I copied from the prod dockerfile